### ISSUE TITLE
Cannot open any notes

### ISSUE THREAD

**Issue by oyse** [nextcloud/notes#384]
**Describe the bug**

I can no longer open any notes after upgrading to Nextcloud 16.0.3 (then later upgraded to 16.0.4)

Looking at the JSON response it looks like this

{"JSON":{"etag":"45c5269c51a890deaf1d02139403a959","modified":null,"title":"Fusion project","category":"statkraft","content":"Error: (Fusion project.txt) An exception occurred while executing 'INSERT INTO \"oc_file_locks\" (\"key\", \"lock\", \"ttl\") VALUES(?, ?, ?) ON CONFLICT DO NOTHING' with params [\"files\\/3f12ddc0287423160f601fa2e1029d01\", 1, 1569178240]:\n\nSQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near \"ON\"\nLINE 1: ..._locks\" (\"key\", \"lock\", \"ttl\") VALUES($1, $2, $3) ON CONFLIC...\n                                                             ^","favorite":false,"error":true,"errorMessage":"Error: (Fusion project.txt) An exception occurred while executing 'INSERT INTO \"oc_file_locks\" (\"key\", \"lock\", \"ttl\") VALUES(?, ?, ?) ON CONFLICT DO NOTHING' with params [\"files\\/3f12ddc0287423160f601fa2e1029d01\", 1, 1569178240]:\n\nSQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near \"ON\"\nLINE 1: ..._locks\" (\"key\", \"lock\", \"ttl\") VALUES($1, $2, $3) ON CONFLIC...\n                                                             ^","id":6234}}

**To Reproduce**
1. Go to Notes app in the webpage
2. Click any note
3. Note does not open, but at the top of the page an error message is seen

**Expected behavior**
That I can open a note

**Server (please complete the following information):**
- Notes app version: 3.0.3
- Nextcloud version: 16.0.4
- OS: entos-release-7-6.1810.2.el7.centos.x86_64
- Web server: Not known. Managed by hosting provider. Probably Apache
- PHP version: 7.1 
- Database: PostgreSQL 9.4.24 on x86_64-unknown-linux-gnu, compiled by gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-36), 64-bit

**Client (please complete the following information):**
 - Browser (incl. version): Firefox 69.0. Also newer Chrome
 - OS: Linux and Windows.

This mostly looks like server side issue.

**Log files**

**Comment by oyse** [comment]
Attached log file. 
[nextcloud-notes-error.log](https://github.com/nextcloud/notes/files/3640034/nextcloud-notes-error.log)

**Comment by korelstar** [comment]
It looks like this is a known issue in Nextcloud server, see https://github.com/nextcloud/server/issues/15441 and https://github.com/nextcloud/server/issues/15613 . Currently, [Nextcloud 16 requires PostgreSQL 9.5 or above](https://docs.nextcloud.com/server/16/admin_manual/installation/system_requirements.html#server); you're using PostgreSQL 9.4.

Nevertheless, it looks like they wrote a fix for this (https://github.com/nextcloud/server/pull/16572) which will be included in Nextcloud 16.0.5 (see https://github.com/nextcloud/server/pull/16814).

To sum up, there is nothing we can do here. Please upgrade to PostgreSQL 9.5+ or wait for [Nextcloud 16.0.5](https://github.com/nextcloud/server/milestone/92).

**Comment by oyse** [comment]
Ok. Thank you for looking into it. I will wait for the 16.0.5 release.
